code/index.html

```diff
@@ -4,27 +4,274 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
+    <link rel="stylesheet" href="main.css" />
   </head>
   <body>
-    <!-- video or image as a header is cool :) -->
+    <!-- Header with full width video & nav links to login -->
+    <header class="hero">
+      <video class="hero__video" playsinline autoplay muted loop>
+        <source src="./assets/videos/adventures.mp4" type="video/mp4" />
+        <source src="./assets/videos/adventures.webm" type="video/webm" />
+        Sadly your browser not support HTML 5 video.
+      </video>
+      <h1 class="hero__title">
+        Let the <br />
```

Comment:
We shouldn't use `<br />` tags - more information on that in [Week 2 Master document](https://www.notion.so/technigo/Week-2-Project-news-site-bbe695289d154ff98d6a247cef873f07)